src/main/java/juja/microservices/keepers/service/KeepersService.java

```diff
@@ -64,6 +64,22 @@ public String addKeeper(KeeperRequest keeperRequest){
     }
 
     public Map<String, List<String>> getActiveKeepers() {
-        return keepersRepository.getActiveKeepers();
+        logger.debug("Service.getActiveKeepers after in, without any parameters.");
+        Map<String, List<String>> outMap = new HashMap<>();
```

Comment:
let's rename this map. I want to be clear what is this map contain. For example, keepersWithDirections.